app/src/main/java/no/ntnu/klubbhuset/data/repository/AdminOrganizationRepository.java

```diff
@@ -119,16 +127,14 @@ private void loadUserPaymentStatus(String email, Club club) {
                         return Response.error(new VolleyError(response));
                     }
                 }
-
-                @Override
-                public Map<String, String> getHeaders() throws AuthFailureError {
-                    return AuthHelper.getAuthHeaders(getApplication());
-                }
             };
             requestQueue.add(request);
-        } catch (JSONException e) {
+        } catch (
+                JSONException e) {
             Log.e(TAG, Objects.requireNonNull(e.getMessage()));
         }
-
+        return userPaymentStatus;
     }
-}
\ No newline at end of file
+
+}
+
```

Comment:
Is this correct ?

Reply:
`BarcodeViewModel` -> `AdminOrganizationRepository`

Reply:
Sannsynligvis, siden appen fungera som den skal